--- xf86-video-s3virge-1.10.7/src/s3v_driver.c~	2015-09-27 01:28:11.000000000 +0900
+++ xf86-video-s3virge-1.10.7/src/s3v_driver.c	2018-05-23 22:00:43.635026154 +0900
@@ -1203,18 +1203,6 @@
    vga256InfoRec.directMode = XF86DGADirectPresent;
 #endif
 
-    /*
-     * xf86ValidateModes will check that the mode HTotal and VTotal values
-     * don't exceed the chipset's limit if pScrn->maxHValue and
-     * pScrn->maxVValue are set.  
-     */
-
-    /* todo -  The virge limit is 2048 vertical & horizontal */
-    /* pixels, not clock register settings. */
-			 	/* true for all ViRGE? */
-  pScrn->maxHValue = 2048;
-  pScrn->maxVValue = 2048;
-
     				/* Lower depths default to config file */
   pScrn->virtualX = pScrn->display->virtualX;
 				/* Adjust the virtualX to meet ViRGE hardware */
